R/defaults.R

```diff
@@ -71,22 +71,27 @@ epidist_stancode.default <- function(data, ...) {
 
 #' Default method used for interface using `brms`
 #'
+#' @inheritParams epidist_formula
 #' @inheritParams epidist
 #' @rdname epidist.default
 #' @method epidist default
 #' @family defaults
 #' @export
-epidist.default <- function(data, formula = epidist_formula(data),
-                            family = epidist_family(data),
-                            prior = epidist_prior(data), fn = brms::brm, ...) {
+epidist.default <- function(data, formula = list(mu ~ 1, sigma ~ 1),
+                            family = brms::lognormal(), fn = brms::brm, ...) {
   epidist_validate(data)
-  stancode <- epidist_stancode(data = data, family = family)
+  epidist_family <- epidist_family(data, family)
+  epidist_formula <- epidist_formula(
+    data = data, family = epidist_family, formula = formula
```

Comment:
Here it's a challenge whether things are `family` as input or `epidist_family` as input (likewise `formula` or `epidist_formula`)